packages/amplication-data-service-generator/src/server/resource/controller/controller.base.template.ts

```diff
@@ -76,13 +60,6 @@ export class CONTROLLER_BASE {
     });
   }
 
-  // @ts-ignore
-  @common.UseInterceptors(AclFilterResponseInterceptor)
-  @nestAccessControl.UseRoles({
-    resource: ENTITY_NAME,
-    action: "read",
-    possession: "any",
-  })
   @common.Get()
   @swagger.ApiOkResponse({ type: [ENTITY] })
   @swagger.ApiForbiddenResponse()
```

Comment:
I think this is also should be removed as `ApiForbiddenResponse` is for unauthorized users.

@yuval-hazaz this means that we also need to remove the "public" logic from the server and the client. Maybe only extract the public decorator to a plugin that runs after the auth plugin if some user wants to have specific public routes